src/md/release.md

```diff
@@ -150,6 +150,47 @@ mkdir svn-dist && cp *-source-* svn-dist/
 svn import svn-dist https://dist.apache.org/repos/dist/dev/apex/${RNAME}-RC1 -m "Apache Apex v${rv}-RC1"
 ```
 
+## Build and Deploy Documentation
+
+The documentation will be generated as static HTML files and copied into the ```apex-site``` repository. It will be available at an alternate URL location until the release is promoted, at which time it will also be promoted to the main website location.
```

Comment:
I don't think we need an alternate location and the steps below don't seem to indicate it either.